.github/CODEOWNERS

```
@@ -83,6 +83,8 @@ Makefile                                              @thiagokokada
 
 /modules/programs/broot.nix                           @aheaume
 
+/modules/programs/btop.nix                            @GaetanLepage
```

Can also add the test directory:

```suggestion
/modules/programs/btop.nix                            @GaetanLepage
/tests/modules/programs/btop                          @GaetanLepage
```

---

done :)